### PR TITLE
Made full mod info show internal name

### LINKED ISSUE #2832: Full mod info needs to include the internal name
as it doesn't seem to show that currently

### PR DESCRIPTION
**Pull request by Generatoror**
**Brief Description of What This PR Does**

Added a label that shows internal mod name on the full mod info popup

**Related Issues**
closes #2832

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [ ] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [ ] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author.

### CODE REVIEW

**Comment by hhyyrylainen** [comment]
You need to update the translations: https://github.com/Revolutionary-Games/Thrive/blob/master/doc/working_with_translations.md#updating-the-localizations

File: src/auto-evo/simulation/PopulationSimulation.cs [hunk 1]
```diff
@@ -31,9 +31,11 @@ public static void Simulate(SimulationConfiguration parameters)
                 patchesToSimulate = patchesToSimulate.Where(p => parameters.PatchesToRun.Contains(p.Value));
             }
 
+            var patchesList = patchesToSimulate.ToList();
+
             while (parameters.StepsLeft > 0)
             {
-                RunSimulationStep(parameters, speciesToSimulate, patchesToSimulate, random, cache);
+                RunSimulationStep(parameters, speciesToSimulate, patchesList, random, cache);
```
Comment (adQuid) [inline src/auto-evo/simulation/PopulationSimulation.cs:38]: Is your fork not fetched recently? This doesn't look related to the PR.

**Comment by hhyyrylainen** [comment]
You can force push over existing PRs. So there is no need to create a new PR to even replace every single commit in the PR. If you can't push normally use force pushing to force overwrite commits in a PR.

**Comment by Generatoror** [comment]
Yeah sorry, i don't have much experience using git

**Comment by Generatoror** [comment]
How should i proceed?

**Comment by hhyyrylainen** [comment]
First you should fix your local version by doing rebases or cherry picking to a new branch. Then I think you should be able to reopen this PR and push to your fork's branch this is from with a force push with that cleaned up branch you make, to overwrite all the commits in this PR.

**Comment by Generatoror** [comment]
Oh i already fixed it, but i don't think i can reopen this pr because i deleted the original branch, I think i have to create a new pr

**Comment by Generatoror** [comment]
Oh i already fixed it, but i don't think i can reopen this pr because i deleted the original branch, I think i have to create a new pr

**Comment by hhyyrylainen** [comment]
What if you push the new branch with the previous name?

**Comment by Generatoror** [comment]
It has the same name, but i can't reopen this pr. i literally deleted it and recreated it

**Comment by hhyyrylainen** [comment]
I guess then you'll have to open a new PR.
